library/std/src/lib.rs

```diff
@@ -171,19 +171,19 @@
 //! [`Result<T, E>`]: result::Result
 //! [`Vec<T>`]: vec::Vec
 //! [`atomic`]: sync::atomic
-//! [`for`]: ../book/ch03-05-control-flow.html#looping-through-a-collection-with-for
+//! [`for`]: https://doc.rust-lang.org/book/ch03-05-control-flow.html#looping-through-a-collection-with-for
```

Comment:
This wouldn't work for offline docs right?

Reply:
You are probably right. Is it possible to do this with intra-doc links, such that it would default to the site if the book isn't included? Does rustdoc even know if the book is included?

Reply:
From what I know, we provide option to either bundle offline docs or without, so there is no option to just bundle the api docs only, I supposed using this method is safe. Is there any use cases where this break for you?